src/Components/Navbar.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import array from './Data_Array'
+import {ReactComponent as Logo} from '../Assets/Logo-text.svg'
+
+const Navbar = () => {
+  function Search() {
```

Comment:
I would suggest that you Javascript higher-order functions for this implementation refer to this tutorial to get a good hang of the usage ( https://www.youtube.com/watch?v=rRgD1yVwIvE&t=509s)